boto/auth.py

```diff
@@ -383,9 +383,9 @@ def query_string(self, http_request):
         parameter_names = sorted(http_request.params.keys())
         pairs = []
         for pname in parameter_names:
-            pval = boto.utils.get_utf8_value(http_request.params[pname])
+            pval = boto.utils.get_binary_str(http_request.params[pname])
             pairs.append(urllib.parse.quote(pname, safe=''.encode('ascii')) +
-                         '='.encode('ascii') +
+                         '=' +
```

Comment:
Curious, in Python 3 where this is a unicode string, will pairs.append coerce it to ascii, or coerce the rest to unicode?